_IBC/cvn-osmosis.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "../ibc_data.schema.json",
+    "chain_1": {
+      "chain_name": "conscious",
```

Comment:
The file name "cvn-osmosis.json" should be renamed to use the chain_name, i.e., "conscious-osmosis.json"